### ISSUE TITLE
Open up/extend the OAI harvester

### ISSUE THREAD

**Issue by menzowindhouwer** [CLARIAH/clariah-plus#56]
Open up/extend OAI harvester:

1. To speak to the CLARIAH endpoint registr(y|ies)
2. To speak the needed protocols
3. To deal with the needed formats

**Comment by ddeboer** [comment]
@menzowindhouwer I heard you mention that you want to connect the harvester to the NDE Register.

I’m not seeing a separate issue for that, so assuming that you referred to this one. Let me know if you and @vicding-mi need any help with building the right SPARQL queries to retrieve dataset descriptions from the [NDE Register SPARQL endpoint](https://triplestore.netwerkdigitaalerfgoed.nl).

**Comment by wmelder** [comment]
How does the NDE register handle updates? Does it also do regular updates from the sources it has registered? Would Clariah then also have to run some SPARQL queries regularly to stay up-to-date? Or is there some kind of queue where changes are pushed?

**Comment by menzowindhouwer** [comment]
@vicding-mi and I are preparing for a pull like approach, i.e. CLARIAH+ regularly reruns the queries at the NDE endpoint ...

**Comment by ddeboer** [comment]
> How does the NDE register handle updates? Does it also do regular updates from the sources it has registered? 

Yep: the NDE Register periodically [crawls registered datasets](https://github.com/netwerk-digitaal-erfgoed/dataset-register#crawler), so updates to the datasets will end up (eventually) in CLARIAH+ as well.

**Comment by wmelder** [comment]
>Yep: the NDE Register periodically [crawls registered datasets](https://github.com/netwerk-digitaal-erfgoed/dataset-register#crawler), so updates to the dataset descriptions will end up (eventually) in CLARIAH+ as well.

In that case we probably better add the B&G datasets to the NDE register using the [API](https://datasetregister.netwerkdigitaalerfgoed.nl/api/static/index.html#/default/post_datasets), so that no additional Clariah harvester/crawler is needed for B&G?

**Comment by ddeboer** [comment]
@vicding-mi @menzowindhouwer I created #97 for information specific to the NDE Dataset Register.
